fuzzers/pip_loop.mk

```diff
@@ -0,0 +1,75 @@
+TODO_N ?= 50
```

Comment:
What does this do differently vs int_loop?

Reply:
int_loop database and pushdb targets are for int_l and int_r tiles only.  This pip fuzzer is for bram_l and bram_r, so it needs to define different database and pushdb targets.  I refactored the common parts of both into pip_loop.mk and int_loop.mk includes pip_loop.mk